src/main/java/com/google/api/generator/gapic/composer/utils/ClassNames.java

```diff
@@ -26,9 +26,21 @@
   private static final String SERVICE_SETTINGS_CLASS_NAME_PATTERN = "%sSettings";
   private static final String SERVICE_STUB_SETTINGS_CLASS_NAME_PATTERN = "%sStubSettings";
   private static final String SERVICE_STUB_CLASS_NAME_PATTERN = "%sStub";
+
+  // TODO: remove after Pre-DIREGAPIC refactoring is fully merged
   private static final String GRPC_SERVICE_STUB_CLASS_NAME_PATTERN = "Grpc%sStub";
+  // TODO: remove after Pre-DIREGAPIC refactoring is fully merged
```

Comment:
The problem is, as you mentioned in the other comment that it can be `gRPC` or `Grpc` and the enum itself is `GRPC`, which is ugly and the whole issue is about just generating a comment string. I guess technically the ClassNames and CommentComposer should repeat the fate of the other composers - have their own hierarchy to match the rest of the code architecturally, I just felt like that was an overkill for such a simple case, that is why went with a simple parameter. We can do something fancier, if you feel like it is worth it. Should we?

Reply:
Resolved with the latest refactoring.